core/src/main/scala/kanaloa/reactive/dispatcher/queue/QueueProcessor.scala

```diff
@@ -21,6 +21,11 @@ class QueueProcessor(
   resultChecker:          ResultChecker
 ) extends Actor with ActorLogging with MessageScheduler {
 
+  val healthCheckSchedule = {
```

Comment:
Hrm...I guess we can choose this approach for now, and see how it behaves.  I'm still concerned about having durations where there are no Workers, so maybe for now just add the Terminated Workers check back in?